Classes/Private/Manager/Submanagers/OCTSubmanagerCallsImpl.m

```diff
@@ -7,7 +7,7 @@
 #import "OCTTox.h"
 
 const OCTToxAVAudioBitRate kDefaultAudioBitRate = OCTToxAVAudioBitRate48;
-const OCTToxAVVideoBitRate kDefaultVideoBitRate = 2000;
+const OCTToxAVVideoBitRate kDefaultVideoBitRate = 2500;
```

Comment:
this is OK as default options, since in my opinion most iPhone users will be on 3g network.

but later this should be a **user changeable** setting:

2500 -> for 3g
500 -> for Wifi (or strong LTE network)

Reply:
I guess app should change this parameter itself based on connection type.